input/pages/profiles.md

```diff
@@ -71,23 +71,23 @@ To support packaging, testing, and distribution of measure and library artifacts
 
 | **Profile** | **Description** | 
 |----|----|
-| CQFMDevice | A software device used in the creation, validation, evaluation, packaging, and/or testing of a library or measure artifact.  |
-| CQFMTestCase | A measure report profile that allows definition and exchange of test cases for a measure.  |
+| [CQFMDevice](http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/device-softwaresystem-cqfm.html) | A software device used in the creation, validation, evaluation, packaging, and/or testing of a library or measure artifact.  |
+| [CQFMTestCase](https://build.fhir.org/ig/HL7/cqf-measures/StructureDefinition-test-case-cqfm.html) | A measure report profile that allows definition and exchange of test cases for a measure.  |
 {: .grid }
 
 
 
 ## Alphabetical Listing
 
```

Comment:
The tracker has comment on this Alphabetical Listing: "This does not appear to be a complete list, e.g., missing CQFMExecutableMeasure" that is not addressed by this PR.

These missing profiles need to not added. 
- CQFM Executable Measure 
- CQFM Publishable Manifest